app/questionnaire/placeholder_parser.py

```diff
@@ -57,8 +58,8 @@ def __init__(
         response_metadata: MutableMapping,
         schema: QuestionnaireSchema,
         renderer: PlaceholderRenderer,
-        progress_store: ProgressStore,
-        supplementary_data_store: SupplementaryDataStore,
+        progress_store: ps.ProgressStore,
+        supplementary_data_store: sds.SupplementaryDataStore,
```

Comment:
Args need removing here too and replacing with data stores

Reply:
Similar issue to https://github.com/ONSdigital/eq-questionnaire-runner/pull/1228#discussion_r1362072105, I will try to refactor this more.

Reply:
Ahhh I see - that makes sense now, i think you can do the imports under `if TYPE_CHECKED` to avoid the circular errors